Integrations/GoogleBigQuery/GoogleBigQuery.py

```diff
@@ -173,9 +173,7 @@ def test_module():
         query_job = bigquery_client.query(TEST_QUERY)
         query_results = query_job.result()
         results_rows_iterator = iter(query_results)
-        first_item = next(results_rows_iterator)
-        if str(first_item.get("name")) != "Ruby":
-            raise ValueError("Data from DB not matching expected results")
+        next(results_rows_iterator)
```

Comment:
try to catch an error here and post a good error msg if an error occurs

Reply:
There shouldn't be an error here, why would there be? It's a huge DB which consistently returns results